Lib/token.py

```diff
@@ -104,13 +110,23 @@ def _main():
     prog = re.compile(
         "#define[ \t][ \t]*([A-Z0-9][A-Z0-9_]*)[ \t][ \t]*([0-9][0-9]*)",
         re.IGNORECASE)
+    comment = re.compile(
+        "^\s*/\*\s*(.+)\s*\*/\s*$",
+        re.IGNORECASE)
+
     tokens = {}
+    prev_val = None
     for line in lines:
         match = prog.match(line)
         if match:
             name, val = match.group(1, 2)
-            val = int(val)
-            tokens[val] = name          # reverse so we can sort them...
+            prev_val = int(val)
+            tokens[prev_val] = {'token': name}          # reverse so we can sort them...
```

Comment:
I would prefer to use val here, but set prev_val after the tokens assignement. Here we use the current value, not the previous value.